src/components/NewActivityList.js

```diff
@@ -0,0 +1,147 @@
+import React, { Component } from 'react'
+import { Table, Button, Modal, ControlLabel, FormControl } from 'react-bootstrap'
+import './components.css'
+import firebase from 'firebase'
+import toastr from 'toastr'
+import moment from 'moment'
+import CountDown from './CountDown'
+
+class NewActivityList extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      hours: '',
+      money: '',
+      how_long: true,
+      startedAt: '',
+      isModalShown: true
+    }
+  }
+
+  componentDidMount() {
+    const userId = firebase.auth().currentUser.uid;
```

Comment:
it's not a mistake at all, but i like this more, it's up to you to decide which one looks better

const { uid }  = firebase.auth().currentUser

if you choose mine, don't forget to replace userId with uid